app/_data/docs_nav_kuma_dev.yml

```diff
@@ -805,3 +807,150 @@ items:
+  - url: /generated/cmd/kumactl/kumactl_get_meshhealthchecks/
+  - url: /generated/cmd/kumactl/kumactl_get_meshhttproute/
+  - url: /generated/cmd/kumactl/kumactl_get_meshhttproutes/
+  - url: /generated/cmd/kumactl/kumactl_get_meshproxypatch/
+  - url: /generated/cmd/kumactl/kumactl_get_meshproxypatches/
+  - url: /generated/cmd/kumactl/kumactl_get_meshratelimit/
+  - url: /generated/cmd/kumactl/kumactl_get_meshratelimits/
+  - url: /generated/cmd/kumactl/kumactl_get_meshretries/
+  - url: /generated/cmd/kumactl/kumactl_get_meshretry/
+  - url: /generated/cmd/kumactl/kumactl_get_meshtimeout/
+  - url: /generated/cmd/kumactl/kumactl_get_meshtimeouts/
+  - url: /generated/cmd/kumactl/kumactl_get_meshtrace/
+  - url: /generated/cmd/kumactl/kumactl_get_meshtraces/
+  - url: /generated/cmd/kumactl/kumactl_get_meshtrafficpermission/
+  - url: /generated/cmd/kumactl/kumactl_get_meshtrafficpermissions/
+  - url: /generated/cmd/kumactl/kumactl_get_proxytemplate/
+  - url: /generated/cmd/kumactl/kumactl_get_proxytemplates/
+  - url: /generated/cmd/kumactl/kumactl_get_rate-limit/
+  - url: /generated/cmd/kumactl/kumactl_get_rate-limits/
+  - url: /generated/cmd/kumactl/kumactl_get_retries/
+  - url: /generated/cmd/kumactl/kumactl_get_retry/
+  - url: /generated/cmd/kumactl/kumactl_get_secret/
+  - url: /generated/cmd/kumactl/kumactl_get_secrets/
+  - url: /generated/cmd/kumactl/kumactl_get_timeout/
+  - url: /generated/cmd/kumactl/kumactl_get_timeouts/
+  - url: /generated/cmd/kumactl/kumactl_get_traffic-log/
+  - url: /generated/cmd/kumactl/kumactl_get_traffic-logs/
+  - url: /generated/cmd/kumactl/kumactl_get_traffic-permission/
+  - url: /generated/cmd/kumactl/kumactl_get_traffic-permissions/
+  - url: /generated/cmd/kumactl/kumactl_get_traffic-route/
+  - url: /generated/cmd/kumactl/kumactl_get_traffic-routes/
+  - url: /generated/cmd/kumactl/kumactl_get_traffic-trace/
+  - url: /generated/cmd/kumactl/kumactl_get_traffic-traces/
+  - url: /generated/cmd/kumactl/kumactl_get_virtual-outbound/
+  - url: /generated/cmd/kumactl/kumactl_get_virtual-outbounds/
+  - url: /generated/cmd/kumactl/kumactl_get_zone/
+  - url: /generated/cmd/kumactl/kumactl_get_zone-ingress/
+  - url: /generated/cmd/kumactl/kumactl_get_zone-ingresses/
+  - url: /generated/cmd/kumactl/kumactl_get_zoneegress/
+  - url: /generated/cmd/kumactl/kumactl_get_zoneegresses/
+  - url: /generated/cmd/kumactl/kumactl_get_zones/
+  - url: /generated/cmd/kumactl/kumactl_uninstall_ebpf/
+  - url: /generated/cmd/kumactl/kumactl_uninstall_transparent-proxy/
+  - url: /generated/cmd/kumactl/kumactl_install_gateway_kong/
+  - url: /generated/cmd/kumactl/kumactl_install_gateway_kong-enterprise/
+  - url: /generated/cmd/kumactl/kumactl_config_control-planes_add/
+  - url: /generated/cmd/kumactl/kumactl_config_control-planes_list/
+  - url: /generated/cmd/kumactl/kumactl_config_control-planes_remove/
+  - url: /generated/cmd/kumactl/kumactl_config_control-planes_switch/
```

Comment:
do we need to manually update this or is it generated? if it's not generated can't we just use `/generated/*` or something similar so we don't have to update it?

Reply:
I manually added them, but that was before realizing that we don't need to single-source the generated files. Hence, we don't need to add the whole list, just the ones we want to see rendered on the sidenav.